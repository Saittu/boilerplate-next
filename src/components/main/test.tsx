import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('renders without crashing', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /Next Boilerplate/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
