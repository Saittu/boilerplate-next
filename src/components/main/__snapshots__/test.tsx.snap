// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> renders without crashing 1`] = `
<div>
  <h1>
    Next Boilerplate
  </h1>
  <p>
    teste
  </p>
</div>
`;
